internal/tools/orchestrator/services/addon/addon_handler.go

```diff
@@ -1149,6 +1149,9 @@ func (a *Addon) basicAddonDeploy(addonIns *dbclient.AddonInstance, addonInsRouti
 	}
 	logrus.Infof("sending addon creating request, request body: %+v", *addonCreateReq)
 
+	_, ok := addonCreateReq.DiceYml.Meta["USE_OPERATOR"]
+	a.pushLog(fmt.Sprintf("use operator: %v", ok), params)
```

Comment:
I think we can add more target addon info, such as version, name,  before submitting the `ServiceGroup`.